### DISCUSSION TITLE
TFT_eSPI has been tested with R61529?

### DISCUSSION THREAD

**Discussion by lepromer** [Bodmer/TFT_eSPI#1756] [General]
hello i wonder, can this library  work with R61259 , Has it been tested?

**Comment by Bodmer** [comment]
The library does not support that display driver.

The supported drivers can be seen here:
https://github.com/Bodmer/TFT_eSPI/tree/master/TFT_Drivers

**Reply by lepromer** [reply]
Thank You bodmer
